src/wiki_resolver.py

```
@@ -0,0 +1,79 @@
+                "redirects": 1,
+                "prop": "pageprops", # <--- ASK FOR PAGE PROPERTIES
+                "format": "json"
+            },
+            headers=HEADERS, timeout=5
+        )
+        data = r.json()
+        
+        # 1. Did we get a redirect? (e.g., UK -> United Kingdom)
+        query = data.get("query", {})
+        redirects = query.get("redirects", [])
+        final_title = redirects[0]["to"] if redirects else term
```

Ruff (0.15.0)

[error] Local variable `final_title` is assigned to but never used

Remove assignment to unused variable `final_title`

(F841)